core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -595,6 +595,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey UNDERFS_OBJECT_STORE_DEFAULT_MODE =
+      new Builder(Name.UNDERFS_OBJECT_STORE_DEFAULT_MODE)
+          .setDefaultValue("0700")
+          .setDescription("Default mode for objects if mode cannot be discovered. Currently "
+              +"implemented only for S3A.")
```

Comment:
hrmmm, this should either be implemented at the object store ufs level, or be renamed to s3a for only s3a ufs. Ideally, it would apply to the object store ufs.

Reply:
renamed to s3a